changelog/1465.trivial.1.rst

```diff
@@ -0,0 +1,3 @@
+An attempt to import `plasmapy` from a version of Python below the
+minimum supported value will now raise an `ImportError` instead of an
+`Exception`.
```

Comment:
```suggestion
Changed the raised exception to `ImportError` (from a general `Exception`)
when attempting to import `plasmapy` from a Python version below the
minimum supported version.
```

Just re-commenting this because the other suggestion was taken up quickly enough that the suggestion was likely resolved before this comment was viewed.

Again, tense in accordance with our changelog guide.